### ISSUE TITLE
🚧 Dev ➾ Refactor Ligo Native tests in taco-shop 

### ISSUE THREAD

**Issue by hu3man** [ecadlabs/taqueria#1325]
### 🚥 Definition of Done

- [X] 🎯 Planning and Design
- [ ] 🏎️ Dev Implementation (PR merged)
- [ ] ✍️ Release Notes Added
- [ ] 🏆 Product Acceptance

### ⚖️ Acceptance Criteria

✨ Functional: 
- [ ] Tests in taco-shop test the contract functionality
- [ ] Tests that test Taqueria functionality removed
 
 📐 Non-Functional: 
- [ ] Tests written from the perspective of a dApp developer
 

### 🔭 Scope

🙅‍♂️ Out of Scope: 
- Ligo native tosts
 
 

### 🧰 Work Description

Currently, the automated tests in the taco-shop scaffold are written from the pov of a Taqueria dev (ie, testing taqueria functionality rather than the functionality of the contract). The tests need to be re-written to demonstrate what a dApp developer's automated tests would be

### 🦄 Sizing

2

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hu3man** [comment]
Hey @claudebarde, Jev mentioned that you were recently working on adding some Ligo native tests to this contract. Can you give us a hand adding them to the `taco-shop` scaffold?

**Comment by claudebarde** [comment]
Yes sure! I already have some tests ready for the contract in this repo: https://github.com/ecadlabs/taqueria-scaffold-nft (just to confirm that's the one we are talking about).
I also made some slight changes to the contract that I will push soon.

**Comment by hu3man** [comment]
Great, thanks @claudebarde! We definitley want to add some additional tests to the nft-scaffold project but I got the projects mixed up. This ticket in particular was for the `taco-shop` scaffold which uses the hello-tacos contract

**Comment by claudebarde** [comment]
Great, the tests for the hello-tacos contract are going to be even easier to write 🙂

**Comment by egarson** [comment]
Blocked, I do not see any of the tasks listed in `taq help`, and get `zod` an `i18next` errors in a stack trace.

**Comment by egarson** [comment]
I have completed refactoring the Ligo unit tests.

**Comment by egarson** [comment]
Integration tests have been updated as well, but waiting on Environment/config changes before final push.
(This comment is a bit late fwiw).
